apps/zui/scripts/merge-mac-release-files.mjs

```diff
@@ -0,0 +1,227 @@
+// Based on https://github.com/electron-userland/electron-builder/issues/5592#issuecomment-2004803764
+import { Octokit } from "octokit";
+import pkg from '../package.json' assert {type: 'json'};
+import { TextDecoder } from 'node:util';
+import fs from 'node:fs';
+import { Readable } from 'node:stream';
+import { Buffer } from 'node:buffer';
+import yaml from 'js-yaml';
+
+const token = process.env.GH_TOKEN
+
+const client = new Octokit({
+  auth: token
+});
+
+// These are derived from settings in package.json so the script will work on
+// both regular Zui and Zui Insiders.
+const OWNER = pkg.repository.split('/')[3];
+const REPO = pkg.repository.split('/')[4];
+const PRODUCT_NAME= pkg.productName.replaceAll(' ', '-');
+const REPO = 'zui';
```

Comment:
You are redefining the REPO variable- I'm not sure how the compiler is allow you to do this.

Reply:
@mattnibs: Thanks for catching! That was indeed a mistake, specifically when bringing changes from my personal fork back to the regular Zui code base. The potential for these kinds of mistakes is why I've been planning on subjecting @nwt to a final round of test builds made from yet another set of fresh personal fork repos using what's now in this PR, so this error would have been caught at that point. But glad to have been able to fix it now!